README.md

```diff
@@ -128,7 +128,7 @@ job('upstreamJob') {
     }
 
     triggers {
-        githubPullRequest {
+        pullRequest {
```

Comment:
Does this name correspond to a variable or something?  Or should it match the name of a class?